solr/core/src/test/org/apache/solr/handler/admin/TestConfigsApi.java

```diff
@@ -17,19 +17,11 @@
 
 package org.apache.solr.handler.admin;
 
-import static org.apache.solr.client.solrj.SolrRequest.METHOD.DELETE;
-import static org.apache.solr.cloud.Overseer.QUEUE_OPERATION;
-import static org.apache.solr.handler.admin.TestCollectionAPIs.compareOutput;
-
-import java.util.Map;
 import org.apache.solr.SolrTestCaseJ4;
-import org.apache.solr.api.ApiBag;
-import org.apache.solr.common.cloud.ZkNodeProps;
-import org.apache.solr.handler.ClusterAPI;
-import org.apache.solr.response.SolrQueryResponse;
 
 public class TestConfigsApi extends SolrTestCaseJ4 {
 
+    /*
```

Comment:
Are there any tests left in this class?

Reply:
Uh, no haha.  But I plan to uncomment these tests before taking the PR forward.

Reply:
In hindsight, the single test here has significant overlap with TestConfigSetsApi, so I think I _am_ going to remove it altogether.